src/control/cmd/dmg/pool.go

```diff
@@ -59,6 +59,7 @@ type PoolCreateCmd struct {
 	ScmSize    string  `short:"s" long:"scm-size" description:"Per-server SCM allocation for DAOS pool (manual)"`
 	NVMeSize   string  `short:"n" long:"nvme-size" description:"Per-server NVMe allocation for DAOS pool (manual)"`
 	RankList   string  `short:"r" long:"ranks" description:"Storage server unique identifiers (ranks) for DAOS pool"`
+	Policy     string  `short:"P" long:"policy" default:"type=io_size" description:"Pool tiering policy"`
```

Comment:
Don't feel particular strongly about it but this is a little irregular passing a key=value string as an argument to --policy, I would be more comfortable having PolicyType PoolCreateCmd struct field, then we can just convert string to typed const early on. Just a suggestion, definitely not blocking.